### ISSUE TITLE
Fix bug added when the same layer is added to both maps with dual maps active

### ISSUE THREAD

**Issue by BobTorgerson** [ua-snap/mapventure#40]
When a layer is active on one of the maps, if you try to activate that same layer on the other map, the first map remains active but the button toggles for the second map. When trying to turn off the second map's layer, it will toggle the active map's layer off while leaving the buttons untoggled.

**Comment by BobTorgerson** [comment]
This has been resolved once PR #56 is merged into master.
